### PR TITLE
Suffix interfaces with "Interface"

### LINKED ISSUE #8: Interface suffix?
[The spec](https://github.com/php-fig/fig-standards/blob/master/proposed/container.md) uses it, the code does not. Is this being debated or has a decision already been taken about whether to add this suffix or not?

### PR DESCRIPTION
**Pull request by greg0ire**
This is what is specified.
Fixes #8

### CODE REVIEW

File: src/NotFoundExceptionInterface.php [hunk 1]
```diff
@@ -8,6 +8,6 @@
 /**
  * No entry was found in the container.
  */
-interface NotFoundException extends ContainerException
+interface NotFoundExceptionInterface extends ContainerException
```
Comment (greg0ire) [inline src/NotFoundExceptionInterface.php]: Forgot to rename the extended class

Comment (greg0ire) [reply src/NotFoundExceptionInterface.php]: I mean interface

**Comment by greg0ire** [comment]
Did a grep for `Exception` to be sure not to miss anything.

**Comment by moufmouf** [comment]
:+1: Thanks a lot @greg0ire !

@mnapoli : Not completely related but looking at the PHPDoc of "has", I realize we forget to put a "@throw" for `ContainerExceptionInterface`. Maybe we should put one here too?

**Comment by mnapoli** [comment]
thanks :+1:

@moufmouf yep!

**Pull request merged**